plugins/protocolbuffers/cpp/v26.1/buf.plugin.yaml

```diff
@@ -7,3 +7,5 @@ output_languages:
   - cpp
 spdx_license_id: BSD-3-Clause
 license_url: https://github.com/protocolbuffers/protobuf/blob/v26.1/LICENSE
+registry:
+  cmake: {}
```

Comment:
@smaye81 - Going ahead and adding this to this plugin since I'm updating the new version of grpc/cpp to specify it too.

Reply:
Perfect, thanks.